src/services/API/Plio.js

```diff
@@ -29,15 +43,9 @@ export default {
   createPlio() {
     // creates a new draft plio
     var newPlioData = {
-      value: {
-        status: "draft",
-      },
-      bucket: "plio-test-data",
+      created_by: 3,
```

Comment:
hardcoded for now
When auth is completed, the user id being passed here will be parametrised

Reply:
change it to 1